src/dex/hashflow/hashflow.ts

```diff
@@ -714,7 +714,7 @@ export class Hashflow extends SimpleExchange implements IDex<HashflowData> {
     } catch (e) {
       if (
         e instanceof Error &&
-        e.message.endsWith('User is restricted from using Hashflow')
+        e.message.includes('User is restricted from using Hashflow')
```

Comment:
I suggest to lower case message and check just for "user is restricted"

Reply:
done